common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -611,7 +611,7 @@ def convert_video_transcript(file_name, content, output_format):
     """
     name_and_extension = os.path.splitext(file_name)
     basename, input_format = name_and_extension[0], name_and_extension[1][1:]
-    filename = '{base_name}.{ext}'.format(base_name=basename.encode('utf8'), ext=output_format)
+    filename = u'{base_name}.{ext}'.format(base_name=basename, ext=output_format)
```

Comment:
Looks good! 

you just need to make double sure that the core util `get_transcript`'s returned `filename` is not expected to be bytes in any of its occurrence.

Reply:
I have double checked all occurrences and it is not expecting filename to be in bytes in any case.

Reply:
Thank you very much!